File: challenges/codeeval/easy/001/victor.ayalamarin.java
package codeeval;

import java.io.*;
import java.util.Scanner;
/**
 *
 * @author VICTOR
 */
public class FizzBuzz {
    public static void main(String[] args) throws IOException {
        int x,y,n;
        //String ruta="C:\\Users\\VICTOR\\Documents\\Aprueba.txt";
        //File file = new File(ruta);
        File file = new File(args[0]);
        try (Scanner leeArc = new Scanner(file)) {
           while (leeArc.hasNextInt()){
                x=(leeArc.nextInt());
                y=(leeArc.nextInt());
                n=(leeArc.nextInt());
                String resp="";
                for(int i=0;n>i;i++){
                    if (((i+1)%x) == 0 & ((i+1)%y)==0){
                        resp=resp+"FB"+" ";
                    }
                    else{
                        if(((i+1)%x) == 0){
                            resp=resp+="F"+" ";
                        }
                        else{
                            if (((i+1)%y)==0){
                               resp=resp+="B"+" ";
                            }
                            else resp=resp+String.valueOf(i+1)+" ";      
                        }
                    }
                }
                System.out.print(resp.trim()+"\n");
            }
            leeArc.close();
        }
    }
}

File: challenges/codeeval/medium/071/victor.ayalamarin.py
__author__ = "VICTOR"
__date__ = "$Dec 6, 2015 7:24:40 AM$"

#file = open(sys.argv[1], 'r')
file = open('C:\\Users\\VICTOR\\Documents\\Aprueba.txt','r')
for line in file:
    linea=line.strip(' \n')
    n=linea.find(';')
    part1 = linea[:n].strip(' ').split(',')
    part2 =linea[n+1:].strip(' ')
    nx=int(part2)
    for i in range(len(part1)):
        naux=int(part1[i])
        part1[i]=naux
    tam=len(part1)
    j=0
    resp=[]
    while tam>=nx:
        tam=tam-nx
        aux=[]
        for x in range(len(resp),(len(resp)+nx)):
            aux.append(part1[x])
        for k in range(len(aux)):
            resp.append(aux.pop())
    if tam !=0:
        rest=(len(part1))-tam
        for m in range (rest,len(part1)):
            resp.append(part1[m])
    cadenres=''
    for a in range (len(resp)):
        cadenres=cadenres+str(resp[a])+','
    print (cadenres.strip(', '))
file.close()
